app/main.py

```diff
@@ -2,0 +1,20 @@
+import datetime
+from app.cafe import Cafe
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    num_maskless = 0
+    for friend in friends:
+        if "vaccine" not in friend:
+            return "All friends should be vaccinated"
+
+        expiration_date = friend["vaccine"].get("expiration_date")
+        if expiration_date < datetime.date.today():
+            return "All friends should be vaccinated"
```

Comment:
This is a code style issue. You are using a non-descriptive error message. Instead of `return 'All friends should be vaccinated'`, you could provide a more detailed message, like `return 'All friends should be vaccinated. The vaccine of friend {friend_name} has expired.'`